### PR TITLE
Brandon v3.9.0 2

### PR DESCRIPTION
**Pull request by branflake2267**
This updates the PlacesService. 

@twistedpair Can you branch off of 3.9.0 so we can both update the v3.9.0 branch as the master for 3.9.0 until where done updating it. I'd like to catch up with master next and then update the v3.9.0 branch

### CODE REVIEW

**Comment by twistedpair** [comment]
See a1830ef87fc49bed5534ddec0e0d9f4bb23edeec.I pulled in all of my changes. Check out #17. I just need to complete what is not crossed out.

**Comment by twistedpair** [comment]
Do you plan to merge this pull or do you want me to?

File: Apis_Google_Maps/src/com/google/gwt/maps/client/placeslib/PlacesService.java [hunk 1]
```diff
@@ -45,12 +46,13 @@ private static final native PlacesService createJso(Element element) /*-{
...
   /**
    * Retrieves details about the Place identified by the given reference.
+   * 
    * @param request
    * @param handler
    */
   public final native void getDetails(PlaceDetailsRequest request, PlaceDetailsHandler handler) /*-{
     var callback = function(result, status) {
-      @com.google.gwt.maps.client.placeslib.PlacesService::processDetailsCallback(Lcom/google/gwt/maps/client/placeslib/PlaceResult;Ljava/lang/String;Lcom/google/gwt/maps/client/placeslib/PlaceDetailsHandler;)(result, status, handler);
+      $entry(@com.google.gwt.maps.client.placeslib.PlacesService::processDetailsCallback(Lcom/google/gwt/maps/client/placeslib/PlaceResult;Ljava/lang/String;Lcom/google/gwt/maps/client/placeslib/PlaceDetailsHandler;)(result, status, handler));
```
Comment (twistedpair) [inline Apis_Google_Maps/src/com/google/gwt/maps/client/placeslib/PlacesService.java:55]: Need to do this all over the place with `$entry()`.

**Comment by branflake2267** [comment]
Thanks :)

**Pull request merged**